src/analyser.cpp

```diff
@@ -261,14 +291,39 @@ bool AnalyserInternalEquation::hasNonConstantVariables()
     return hasNonConstantVariables(mVariables) || hasNonConstantVariables(mOdeVariables);
 }
 
-bool AnalyserInternalEquation::check(size_t &equationOrder, size_t &stateIndex,
-                                     size_t &variableIndex,
-                                     const AnalyserModelPtr &model)
+bool AnalyserInternalEquation::variableOnLhsRhs(const AnalyserInternalVariablePtr &variable,
+                                                bool lhs)
```

Comment:
I think I would have just passed the AST child to this function to test if it was on that side or not.

Reply:
ef180f21